docs/keywords/Governance

```diff
@@ -0,0 +1,5 @@
+Data governance plugins
```

Comment:
Capitalize the first letter of each word to maintain consistency throughout.

`Data Governance Plugins`